### ISSUE TITLE
ability to permanently allocate a closure

### ISSUE THREAD

**Issue by plicease** [plicease/FFI-Platypus#9]
The declarative syntax should be something like this:

```
  my $closure = sticky closure { ... }
```

`sticky` can just be a function (xsub or Perl) that increments the ref count.  Now you can pass it into a function that expects it without having to save the code ref anywhere.

```
  foo(1,2, sticky closure { ... } , 4, 5, ..);
```

**Comment by plicease** [comment]
The other way to do this is to save the coderef internally.

I don't think we need an OO interface for this.  If the user needs that ability he can save the code ref himself.

**Comment by plicease** [comment]
must fix #4 first of course.

**Comment by plicease** [comment]
Actually do not have to fix #4, without `sticky` you still get free'd sub ref errors.
